src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -23,6 +23,10 @@
     /** The directory where JSON files used to create data bundles are stored. */
     public static final String TEST_DATA_FOLDER = "src/test/resources/data";
 
+    /** The directory where credentials used in Gmail API are stored. */
+    @SuppressWarnings("WeakerAccess")
```

Comment:
Well because it is currently only accessed in the package `teammates.test.driver` there is a warning.

The other two fields that have these warnings are `TEST_STUDENT2_PASSWORD` and `BACKDOOR_KEY`.